Makefile

```diff
@@ -22,7 +22,7 @@ COVERPKG := $(shell go list ./...  | grep -v '/cmd/' | egrep -v '(testing|test|m
 # Used for e2e test
 export GO_IMAGE = golang:$(shell go version | cut -d ' ' -f 3 | tail -c +3 )
 
-conduit: go-algorand
+conduit: cmd/algorand-indexer/algorand-indexer go-algorand
```

Comment:
This isn't a real dependency is it? If you want `make` to build `algorand-indexer` move this target down. The default target is the first one.

Alternatively, have the first target be "all".